marketplacetests/marketplace/pages/base.py

```diff
@@ -137,6 +129,18 @@ def show_menu(self):
             lambda m: 'mkt-nav--visible' in body.get_attribute('class'))
         return NavMenu(self.marionette)
 
+    def search_for_paid_apps(self):
+        self.set_region('us')
+        return self.search(':paid')
+
+    def is_app_installed(self, name):
```

Comment:
Thinking about it, this probably also belongs to `MarketplaceGaiaTestCase` as it doesn't use the page/app object.